### PR TITLE
Ie8 images

### PR DESCRIPTION
**Pull request by NataliaLKB**
Going part way to solving https://github.com/guardian/frontend/issues/9844 and finishing [this](https://trello.com/c/qv21Bc9w/38-ie8-no-images-for-1-of-users) story in Trello. 

The problem was that only the first few images were loading in IE8. This was caused by two separate exceptions. The first was the use of `addEventListener` which doesn't work in IE8 or older browsers. The 2nd was the use of a reserved word `default`.

Now Images load fine...

image

### CODE REVIEW

File: common/app/templates/headerInlineJS/bootCurl.scala.js [hunk 1]
```diff
@@ -99,6 +99,11 @@ require([
         }
     };
 
+    // IE8 and below use attachEvent
+    if (!window.addEventListener) {
```
Comment (NataliaLKB) [inline common/app/templates/headerInlineJS/bootCurl.scala.js]: https://msdn.microsoft.com/en-us/library/ms536343(v=vs.85).aspx

File: common/app/templates/headerInlineJS/bootCurl.scala.js [hunk 2]
```diff
@@ -99,6 +99,11 @@ require([
...
+        window.addEventListener = window.attachEvent;
+    }
+
     // Report unhandled promise rejections
     // https://github.com/cujojs/when/blob/master/docs/debug-api.md#browser-window-events
```
Comment (gklopper) [inline common/app/templates/headerInlineJS/bootCurl.scala.js]: Inside Scala templates if you do comments like this then they get stripped out at compile time and don't end up in the page source...

`@*  this is a comment  *@`

File: common/app/templates/headerInlineJS/bootCurl.scala.js (hunk 1 above)
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/bootCurl.scala.js]: Ooo

**Comment by gklopper** [comment]
:+1:

**Comment by OliverJAsh** [comment]
Does this mean we can close https://github.com/guardian/frontend/issues/9844? Was the error detailed in https://github.com/guardian/frontend/issues/9844 related to this, or separate? Is it still happening?

BTW you can automatically close GitHub issues when commits are merged by mentioning them in your commit message: https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by NataliaLKB** [comment]
@OliverJAsh Images load fine, but there is a lot more work to be done before any of the ads will load. We could close https://github.com/guardian/frontend/issues/9844 and open a new one simply for that .

**Pull request merged**